src/lib-components/forms/DateTimeInput.vue

```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
```

Comment:
This is effectively a copy/paste of `BaseInput` that drops the `type` prop and any other special type specific handling.  I'd rather not bother with FlatPickr integration just yet, let's talk about whether `DateRangePicker` should adopt a similar date string handling and bite them off at the same time (potentially).